rdmo/views/templatetags/view_tags.py

```diff
@@ -111,6 +111,33 @@ def get_set(context, attribute, set_prefix='', project=None):
     return get_sets(context, attribute, set_prefix=set_prefix, project=project)
 
 
+@register.simple_tag
+def render_custom_inline_list(values=None, sep=',', sep_last=', and'):
```

Comment:
I would call the tag differently, all the `render_` tags have more or less the same signature. I also dislike custom (filler word). Maybe `create_inline_list` or `join_list` / `join_values`?

Reply:
Hmm I like `join_values` more I think.

Reply:
Also `seperator` is better than `sep`.

Reply:
```suggestion
def join_values(values=None, separator=',', sep_last=', and'):
```

Reply:
yes, naming still needed to be discussed.
* `create_inline_list_with_separator`
* `join_values_with_separator`
* `join_values_inline_with_separator`
?

Reply:
-> `join_values_inline`

Reply:
`join_values_inline(values=None, seperator=',', seperator_two='and', seperator_last=', and')`